diff --git a/node_modules/@dump247/storybook-state/index.d.ts b/node_modules/@dump247/storybook-state/index.d.ts
index 68c1894..3ab98b3 100644
--- a/node_modules/@dump247/storybook-state/index.d.ts
+++ b/node_modules/@dump247/storybook-state/index.d.ts
@@ -1,4 +1,4 @@
-import { Renderable, RenderFunction } from '@storybook/react';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
 export type Store<T> = {
   /**
    * @description
@@ -20,8 +20,8 @@ export type Store<T> = {
   reset(): void;
 };
 
-export type LegacyStorybookStateCallback<T> = (store: Store<T>) => Renderable | Renderable[];
+export type LegacyStorybookStateCallback<T> = (store: Store<T>) => StoryFnReactReturnType;
 export function withState<T extends {}>(state: T, callback: LegacyStorybookStateCallback<T>): any // RenderFunction;
 
-export type StorybookStateCallback<T> = (context: {store: Store<T>}) => Renderable | Renderable[];
+export type StorybookStateCallback<T> = (context: {store: Store<T>}) => StoryFnReactReturnType;
 export function withState<T extends {}>(state: T): (callback: StorybookStateCallback<T>) => any
