diff --git a/node_modules/jest-emotion/types/index.d.ts b/node_modules/jest-emotion/types/index.d.ts
index 86936ff..d788bb5 100644
--- a/node_modules/jest-emotion/types/index.d.ts
+++ b/node_modules/jest-emotion/types/index.d.ts
@@ -33,7 +33,7 @@ export default serializer
 
 declare global {
   namespace jest {
-    interface Matchers<R> {
+    interface Matchers<R, T> {
       toHaveStyleRule(
         property: string,
         value: any,
