diff --git a/node_modules/react-spring/renderprops.js b/node_modules/react-spring/renderprops.js
index 489a139..d707ded 100644
--- a/node_modules/react-spring/renderprops.js
+++ b/node_modules/react-spring/renderprops.js
@@ -4,8 +4,8 @@ Object.defineProperty(exports, '__esModule', { value: true });
 
 function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
 
-var _objectWithoutPropertiesLoose = _interopDefault(require('@babel/runtime/helpers/esm/objectWithoutPropertiesLoose'));
-var _extends = _interopDefault(require('@babel/runtime/helpers/esm/extends'));
+var _objectWithoutPropertiesLoose = _interopDefault(require('@babel/runtime/helpers/objectWithoutPropertiesLoose'));
+var _extends = _interopDefault(require('@babel/runtime/helpers/extends'));
 var React = require('react');
 var React__default = _interopDefault(React);
 var ReactDOM = _interopDefault(require('react-dom'));
